### ISSUE TITLE
GPG key import receive failed: No data

### ISSUE THREAD

**Issue by SayidHosseini** [operator-framework/operator-sdk#4435]
#### What did you do?
Following the [installation instructions](https://sdk.operatorframework.io/docs/installation/), tried to import the GPG key with the command `gpg --recv-keys 052996E2A20B5C7E`.

#### What did you expect to see?
I expected the key to be imported and get the following message:
```
gpg: key 052996E2A20B5C7E: public key "Operator SDK (release) <cncf-operator-sdk@cncf.io>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

#### What did you see instead? Under which circumstances?
I got:

```
gpg: keyserver receive failed: No data
```

#### Environment
Ubuntu 20.04.1 LTS

#### Possible Solution
Importing with the `Ubuntu Key Server` fixes it:

```
$ gpg --keyserver keyserver.ubuntu.com --recv-keys 052996E2A20B5C7E
gpg: key 052996E2A20B5C7E: public key "Operator SDK (release) <cncf-operator-sdk@cncf.io>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

#### Additional context
It is noteworthy to say that I had access to the `keys.openpgp.org` server on this ubuntu instance.

**Comment by estroz** [comment]
This seems like an environment issue because others reported being able to pull the key from their default keyserver. The docs should be updated to specifically point to `keyserver.ubunut.com`.

/kind documentation

**Comment by SayidHosseini** [comment]
I also tried it with multiple other linux distros and the same thing there. Maybe it's a linux thing.

**Comment by estroz** [comment]
I was able to pull using the default keyserver on fedora. What gpg version are you using? Mine is `gpg (GnuPG) 2.2.20`.

**Comment by SayidHosseini** [comment]
I'm using the `gpg (GnuPG) 2.2.19` and it is the latest available in the Ubuntu 20.04 repositories.
I didn't try the fedora dist. Can you try with verbose to check what is the keyserver you are receiving from?

I tried with the following distributions and I get the same _**gpg: keyserver receive failed: No data**_:
*  **Debian 10** with `gpg (GnuPG) 2.2.12`
*  **Kali 2020.04** with `gpg (GnuPG) 2.2.20`

However, with the following distributions and I get _**gpg: keyserver receive failed: General error**_:
*  **Ubuntu 18.04.5 LTS** with `gpg (GnuPG) 2.2.4`
*  **Alpine 3.12** with `gpg (GnuPG) 2.2.23`
*  **openSUSE Leap 15.2** with `gpg (GnuPG) 2.2.5`

**Comment by estroz** [comment]
Now this is failing for me too, when it wasn't before :raised_eyebrow:.
